index.html

```diff
@@ -6,13 +6,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>halogen</title>
     <link rel="stylesheet" href="styles.css"></link>
-    <script src="app.js"></script>
+    <script defer src="app.js"></script>
```

Comment:
I like having defer in here! Is the site problematic without it though?

Reply:
I suppose not! I was just taught that if the js file is listed in the head, it's a good idea to write defer there so that the document can finish loading first. But I'm new to all of this

Reply:
The only reason I thought we could leave it out is to see a real example of what it solves when we run into an issue without it haha. so leave it in there!